packages/vite/src/client/client.ts

```diff
@@ -364,6 +364,7 @@ export function updateStyle(id: string, content: string): void {
     if (!style) {
       style = document.createElement('style')
       style.setAttribute('type', 'text/css')
+      style.setAttribute('data-vite-id', id)
```

Comment:
```suggestion
      style.setAttribute('data-vite-dev-id', id)
```

We discussed this and decided to prefix `dev` too so users know it's a dev only feature.